### PR TITLE
Fix issue29: skip the log output of EOF error

### PR DESCRIPTION
**Pull request by gaoxinge**
Fix https://github.com/dubbogo/getty/issues/29.

### CODE REVIEW

File: session.go [hunk 1]
```diff
@@ -720,6 +721,11 @@ func (s *session) handleTCPPackage() error {
 				if netError, ok = perrors.Cause(err).(net.Error); ok && netError.Timeout() {
 					break
 				}
+				if perrors.Cause(err) == io.EOF {
+					err = nil
```
Comment (wongoo) [inline session.go]: `err = nil` is not needed

Comment (gaoxinge) [reply session.go]: @wongoo https://github.com/dubbogo/getty/blob/master/session.go#L654-L657 still log output the EOF error, so I reset `err` to `nil`.

Comment (wongoo) [reply session.go]: get it

**Comment by wongoo** [comment]
LGTM

**Pull request merged**